src/reason_lexer.mll

```diff
@@ -112,6 +112,7 @@ let keyword_table =
     "match", MATCH; (* Including MATCH for source transforming compat *)
     "method", METHOD;
     "module", MODULE;
+    "pub", PUB;
```

Comment:
I think we need to make suer `pub` and `pri` can be valid identifiers. Imagine converting the following code from `ML`:

```
let pri = "private"
```